lib/src/windows/file_picker_windows.dart

```diff
@@ -94,78 +95,94 @@ class FilePickerWindows extends FilePicker {
     String? dialogTitle,
     bool lockParentWindow = false,
     String? initialDirectory,
-  }) {
+  }) async {
+    return compute(_getDirectoryPathIsolate, {
+      'dialogTitle': dialogTitle,
+      'initialDirectory': initialDirectory,
+    });
+  }
+
+  String? _getDirectoryPathIsolate(Map<String, dynamic> args) {
```

Comment:
```suggestion
  String? _getDirectoryPathIsolate(Map<String,Object?> args) {
    String? dialogTitle = args['dialogTitle'] as String?;
    String? initialDirectory = args['initialDirectory'] as String?;
```

Super minor nit: I'd like to avoid `dynamic`, as it disables type-checking. Prefer `Object?` instead.

Reply:
i can constraint to String? for now as it’s the only type of value passed as parameters